ui/admin/tests/acceptance/host-catalogs/host-sets/hosts-test.js

```diff
@@ -28,6 +28,12 @@ module('Acceptance | host-catalogs | host-sets | hosts', function (hooks) {
   setupMirage(hooks);
 
   let getHostSetHostCount;
+  const MANAGE_DROPDOWN_SELECTOR =
+    '[data-test-manage-dropdown-host-sets] div:first-child button';
+  const CREATE_AND_ADD_HOSTS_SELECTOR =
+    '[data-test-manage-dropdown-host-sets] div ul li a';
```

Comment:
Since you did second child for the `ADD_EXISTING_HOSTS_SELECTOR`, should you have made this one specific to the first child? Since technically this grabs both links (but depending on the usage will just return the first one)

```suggestion
    '[data-test-manage-dropdown-host-sets] div ul li:first-child a';
```

Reply:
Ah! Went ahead and made it more explicit;

My thinking was that since I already defined one explicitly, the logic would give the other one implicitly.